### ISSUE TITLE
Tutorials missing workflows

### ISSUE THREAD

**Issue by shiltemann** [galaxyproject/training-material#1459]
This is a list of tutorials that are missing a workflow, if anybody would like to run this tutorial and extract the workflow from their history afterwards and upload it here, this would be a great help for us!

- [ ] **topic:** Introduction, **tutorial:** galaxy-intro-101 [link](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-101/tutorial.html)
- [ ] **topic:** Introduction, **tutorial:** galaxy-intro-peaks2genes [link](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/galaxy-intro-peaks2genes/tutorial.html)
- [ ] **topic:** Genome Annotation, **tutorial:** genome-annotation [link](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html)
- [ ] **topic:** Ecology, **tutorial:** regionalGAM [link](https://training.galaxyproject.org/training-material/topics/ecology/tutorials/regionalGAM/tutorial.html)
- [ ] **topic:** Ecology, **tutorial:** species-distribution-modeling [link](https://training.galaxyproject.org/training-material/topics/ecology/tutorials/species-distribution-modeling/tutorial.html)
- [ ] **topic:** Statistics and machine learning  **tutorial:** machinelearning  [link](https://training.galaxyproject.org/training-material/topics/statistics/tutorials/machinelearning/tutorial.html) 
- [ ] **topic:** Transcriptomics, **tutorial:** rna-seq-viz-with-volcanoplot [link](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-viz-with-volcanoplot/tutorial.html)
- [ ] **topic:** Variant Analysis, **tutorial** non-dip [link](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/non-dip/tutorial.html)
- [ ] **topic:** Variant Analysis, **tutorial** somatic-variants [link](https://training.galaxyproject.org/training-material/topics/variant-analysis/tutorials/somatic-variants/tutorial.html)

And if you run into any other issues or questions when doing the tutorial, please add a fix or file an issue for it :)

**Comment by chrisbarnettster** [comment]
Trying out the Statistics and machine learning  **tutorial**.

**Comment by erasche** [comment]
Updated with fresh data, ca 75% of tutorials have workflows so that's great! :)

**Comment by erasche** [comment]
@Delphine-L would you have a history where you could extract the tn-seq workflow? I have a user who wants to run that tutorial and want to make sure I've collected all the needed tools for it.

**Comment by malloryfreeberg** [comment]
Hi @hexylena @shiltemann !

I have created a workflow for the tutorial "[tal1-binding-site-identification](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/tal1-binding-site-identification/tutorial.html)", and I have identified some updates to the existing tutorial that I am happy to put in a PR. I noticed that at the top of the tutorial it says "Under Development! This tutorial is not in its final state. The content may change a lot in the next months. Because of this status, it is also not listed in the topic pages.". So, I have two questions:
1. Does the TAL1 tutorial still need a workflow?
1. Would it be useful for me to make a PR with updates to the tutorial itself, or is someone else working on it?

Thanks!
#Hacktoberfest

**Comment by hexylena** [comment]
Hi @malloryfreeberg I'm not sure anyone is currently working on it. It looks like it hasn't seen any updates in some time. If you have some updates to the tutorial itself to bring it up to quality, then that's fantastic! We can maybe set `enable: true` finally :) It looks like it still needs a WF so that would be great!

**Comment by malloryfreeberg** [comment]
Thanks @hexylena ! I have made the PR updating the TAL1 tutorial (https://github.com/galaxyproject/training-material/pull/2047) including adding a workflow and updating `enabled: true`.

I am happy to keep going through the tutorials listed in this ticket and generate workflows. The interface has changed quite a bit since I last used Galaxy - it looks great!

**Comment by malloryfreeberg** [comment]
I was going to generate a workflow for [scrna-umis](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/scrna-umis/tutorial.html) but realised there was no way to include a pasted text as input in the workflow-test.yml. I opened a ticket on the main Galaxy repo (https://github.com/galaxyproject/galaxy/issues/10415) and am happy to finish the workflow for this tutorial as a test case when this issue is addressed.

**Comment by malloryfreeberg** [comment]
Is it actually possible to make workflows for the R-based tutorials?
- Tutorial [r-basics](https://training.galaxyproject.org/training-material/topics/introduction/tutorials/r-basics/tutorial.html)
- Tutorial: [rna-seq-counts-to-viz-in-r](https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-counts-to-viz-in-r/tutorial.html)

**Comment by hexylena** [comment]
Nope! Not currently. Maybe someday thanks to new ITs which are workflow compatible, we'll get that, but, not yet.

**Comment by Alaa-Elghamry** [comment]
@bebatut Hi! I think I can help with this 
also, this list needs updating, some of the unchecked tutorials already have workflows
